python/pyspark/sql/tests.py

```diff
@@ -2504,6 +2504,10 @@ def test_conf(self):
         spark.conf.unset("bogo")
         self.assertEqual(spark.conf.get("bogo", "colombia"), "colombia")
 
+        self.assertRaisesRegexp(Exception, "hyukjin", lambda: spark.conf.get("hyukjin"))
+        self.assertEqual(spark.conf.get("hyukjin", None), None)
+        self.assertEqual(spark.conf.get("spark.sql.sources.partitionOverwriteMode", None), None)
```

Comment:
Can you add one more test to get `"spark.sql.sources.partitionOverwriteMode"` without `None` to see the difference between with and without `None`? Hopefully with a simple comment.

Reply:
Sure.